frameworks/hdfs/src/main/java/com/mesosphere/sdk/hdfs/scheduler/Main.java

```diff
@@ -84,7 +86,7 @@ private static String renderTemplate(File configFile, String serviceName) throws
         // Simulate the envvars that would be passed to a task. We want to render the config as though it was being done
         // in a task. Manually copy over a couple envvars which is included in tasks by default.
         Map<String, String> env = new HashMap<>(new TaskEnvRouter().getConfig("ALL"));
-        env.put(EnvConstants.FRAMEWORK_HOST_TASKENV, EndpointUtils.toAutoIpDomain(serviceName));
+        env.put(EnvConstants.FRAMEWORK_HOST_TASKENV, EndpointUtils.toAutoIpDomain(serviceName, schedulerConfig));
```

Comment:
Out of scope of this PR: It seems strange to have these environment variables set here as well as in https://github.com/mesosphere/dcos-commons/pull/2405/files#diff-01c250e232e8ece8351364fa59a4cfd5R398

Reply:
It's because they aren't currently injected into the scheduler, and this requires rendering a template.

Reply:
(Arguably we could always inject them into the scheduler)